.github/workflows/VersionVigilante.yml

```diff
@@ -0,0 +1,34 @@
+name: VersionVigilante
```

Comment:
What's this?

Reply:
screams if you:
1. forget to update the version
2. updated the version twice or more over the master